resources/cert-manager/che-cluster-issuer.yml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+    name: che-cluster-issuer
+    namespace: cert-manager
+spec:
+   ca:
```

Comment:
This file seems not to be well formatted with spaces.

Reply:
Yes, thanks, will align it correctly